applications/script.js

```diff
@@ -6,6 +6,8 @@ import {
   updateApplication,
   getApplicationById,
 } from './utils.js';
+let totalApplicationsRendered = 0;
```

Comment:
- why do we need to keep a count of the number of applications rendered?

Reply:
because we are rendering 6 cards initially and then more 6 after scrolling and so on. 
When i didn't used this indexes were repeating after first six and again and again.
for e.g: 23,22,21,20,19,18,23,22,....